API_SPECS_DEVICE_MODELS_V2.md

```diff
@@ -0,0 +1,459 @@
+            "humidity": 70,
+            "humidity_unit": "%",
+            "pressure": 150,
+            "pressure_unit": "psig",
+            "simulation_state": "normal_pressure"
+        },
+        "Interval": "00:00:10",
+        "Scripts": [
+            {
+                "Type": "javascript",
+                "Path": "chiller-01-state.js"
+            }
+        ]
+    },
+    "Properties": {},
+    "Telemetry": [
+        {
+            "Interval": "00:00:10",
+            "MessageTemplate": "{\"temperature\":${temperature},\"temperature_unit\":\"${temperature_unit}\",\"humidity\":${humidity},\"humidity_unit\":\"${humidity_unit}\",\"pressure\":${pressure},\"pressure_unit\":\"${pressure_unit}\"}",
+            "MessageSchema": {
+                "Name": "chiller-sensors;v1",
+                "Format": "JSON",
+                "Fields": {
+                    "temperature": "Double",
+                    "temperature_unit": "Text",
+                    "humidity": "Double",
+                    "humidity_unit": "Text",
+                    "pressure": "Double",
+                    "pressure_unit": "Text"
+                }
+            }
+        }
+    ],
+    "CloudToDeviceMethods": {},
+}
+```
+
+Response:
+```
+200 OK
+Content-Type: application/json; charset=utf-8
+```
+```json
+{
+    "ETag": "00000000-000000-0000-0000-000000",
+	"Id": "00000000-000000-0000-0000-000000",
+	"Version": "0.0.1",
+	"Name": "Chiller",
+	"Description": "Chiller with external temperature, humidity and pressure sensors.",
+	"Protocol": "MQTT",
+	"Type": "",
+	"Simulation": {
+		"InitialState": {
+			"online": true,
+			"temperature": 75,
+			"temperature_unit": "F",
+			"humidity": 70,
+			"humidity_unit": "%",
+			"pressure": 150,
+			"pressure_unit": "psig",
+			"simulation_state": "normal_pressure"
+		},
+		"Interval": "00:00:10",
+		"Scripts": [
+			{
+				"Type": "javascript",
+				"Path": "chiller-01-state.js"
+			}
+		]
+	},
+	"Properties": {},
+	"Telemetry": [
+		{
+			"Interval": "00:00:10",
+			"MessageTemplate": "{\"temperature\":${temperature},\"temperature_unit\":\"${temperature_unit}\",\"humidity\":${humidity},\"humidity_unit\":\"${humidity_unit}\",\"pressure\":${pressure},\"pressure_unit\":\"${pressure_unit}\"}",
+			"MessageSchema": {
+				"Name": "chiller-sensors;v1",
+				"Format": "JSON",
+				"Fields": {
+					"temperature": "Double",
+					"temperature_unit": "Text",
+					"humidity": "Double",
+					"humidity_unit": "Text",
+					"pressure": "Double",
+					"pressure_unit": "Text"
+				}
+			}
+		}
+	],
+	"CloudToDeviceMethods": {},
+	"$metadata": {
+        "$type": "DeviceModel;2",
+        "$uri": "/v2/devicemodels/00000000-000000-0000-0000-000000",
+        "$created": "2018-03-29T23:33:03+00:00",
+        "$modified": "2018-03-29T23:33:03+00:00"
+    }
+}
+```
+
+## Modifying a device model
+
+### Modifying stock device models
+
+Directly modify the configuration files stored in the file system.
```

Comment:
It would be helpful to give a path to this information or a link to our documentation explaining what is in a device model:
https://docs.microsoft.com/en-us/azure/iot-suite/iot-suite-remote-monitoring-test#define-the-characteristics-of-the-new-device-type